pipelines/ocw/scripts/pipeline-ocw-www.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+SITE_BRANCH=$1  # Should be "preview" or "release"
+if [[ $SITE_BRANCH == "preview" ]]
+then
+  FASTLY_SERVICE_ID=$FASTLY_SERVICE_ID_PREVIEW
+  FASTLY_API_TOKEN=$FASTLY_API_TOKEN_PREVIEW
+  OCW_CONTENT_BUCKET=$OCW_CONTENT_BUCKET_PREVIEW
+elif [[ $SITE_BRANCH == "release" ]]
+then
+  FASTLY_SERVICE_ID=$FASTLY_SERVICE_ID_RELEASE
+  FASTLY_API_TOKEN=$FASTLY_API_TOKEN_RELEASE
+  OCW_CONTENT_BUCKET=$OCW_CONTENT_BUCKET_RELEASE
+else
+  echo "Invalid branch $1"
+  exit 1
+fi
+
+yes | fly -t $FLY_TARGET set-pipeline \
+-p ocw-www-pipeline-$SITE_BRANCH \
+--team=$FLY_TEAM \
+--config=pipelines/ocw/pipeline-ocw-www.yml \
+-v artifact-bucket=$ARTIFACT_BUCKET \
```

Comment:
The artifact bucket should always be the same, so no need to template that, unless you're including the sub-path here?